relay/sources/rudder-package/src/lib.rs

```diff
@@ -97,17 +105,28 @@ pub mod action {
     use tar::Archive;
 
     use crate::archive::Rpkg;
+    use crate::cli::Format;
     use crate::database::Database;
     use crate::repo_index::RepoIndex;
     use crate::repository::Repository;
     use crate::versions::RudderVersion;
     use crate::webapp::Webapp;
     use crate::{
-        PACKAGES_DATABASE_PATH, REPOSITORY_INDEX_PATH, RUDDER_VERSION_PATH, TMP_PLUGINS_FOLDER,
+        PACKAGES_DATABASE_PATH, REPOSITORY_INDEX_PATH, RUDDER_VERSION_PATH, TMP_PLUGINS_FOLDER, list,
     };
     use std::fs::File;
     use std::path::{Path, PathBuf};
 
+    pub fn list(all: bool, enabled: bool, format: Format, webapp: &Webapp) -> Result<()> {
+        // Installed plugins
+        let db = Database::read(PACKAGES_DATABASE_PATH)?;
+        // Available plugins
+        let index = RepoIndex::from_path(REPOSITORY_INDEX_PATH)?;
```

Comment:
We have clients that do not have any repository index available, we should not bail here